GeometryDashAPI/Data/GameData.cs

```diff
@@ -39,26 +39,24 @@ public virtual void Load()
             GC.Collect();
         }
 
+        public virtual bool TrySave(bool checkRunningGame, string fullName = null)
+        {
+            if (checkRunningGame && GameProcess.GameCount() > 0)
+                return false;
+
+            return this.TrySave(fullName);
+        }
+
         public virtual bool TrySave(string fullName = null)
         {
             //Plist > ToString > GetBytes > Gzip > Base64 > Replace > GetBytes > XOR > File
             byte[] gzipc = Crypt.GZipCompress(Encoding.ASCII.GetBytes(Plist.PlistToString(this.DataPlist)));
             string base64 = GameConvert.ToBase64(gzipc);
-            if (fullName == null)
-                File.WriteAllBytes(this.GameDataFile,  Crypt.XOR(Encoding.ASCII.GetBytes(base64), 0xB));
-            else
-                File.WriteAllBytes(fullName, Crypt.XOR(Encoding.ASCII.GetBytes(base64), 0xB));
+
+            File.WriteAllBytes(fullName ?? this.GameDataFile, Crypt.XOR(Encoding.ASCII.GetBytes(base64), 0xB));
             return true;
         }
 
-        public virtual bool TrySave(bool checkRunningGame, string fullName = null)
-        {
-            if (checkRunningGame)
-            {
-                if (GameProcess.GameCount() > 0)
-                    return false;
-            }
-            return this.TrySave(fullName);
-        }
+        public virtual bool TrySave() => this.TrySave(GameDataFile);
```

Comment:
This method `bool TrySave()` confusing compilers, IDEs and brains.  
Because method `bool TrySave(string fullName = null)` exists.  
That is, previously it was already possible to call TrySave without specifying a path

Reply:
I'll complete this in `dev` branch